src/Resources/EmbeddedResources/Paylater/InstallmentPlan.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * Resource representing the installment plan for Paylater Installment.
+ *
+ * Copyright (C) 2023 - today Unzer E-Com GmbH
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  @link  https://docs.unzer.com/
+ *
+ *  @package  UnzerSDK
+ *
+ */
+
+namespace UnzerSDK\Resources\EmbeddedResources\Paylater;
+
+use UnzerSDK\Adapter\HttpAdapterInterface;
+use UnzerSDK\Resources\AbstractUnzerResource;
+use stdClass;
+
+class InstallmentPlan extends AbstractUnzerResource
+{
+    /** @var int $numberOfRates */
+    private $numberOfRates;
+
+    private $totalAmount;
+
+    private $nominalInterestRate;
+
+    /** @var float $effectiveInterestRate */
+    private $effectiveInterestRate;
```

Comment:
This is not even mentioned in the tech spec. !!

Reply:
same as for nominalInterestRate